assets/components/svg/svg.jsx

```diff
@@ -50,6 +50,22 @@ function SvgArrowRightStraight() {
 
 }
 
+// A left-pointing version of the CTA arrow
+function SvgArrowLeftStraight() {
+
+  return (
+    <svg
+      className="svg-arrow-left-straight"
+      xmlns="http://www.w3.org/2000/svg"
+      viewBox="0 0 20 17.89"
+      preserveAspectRatio="xMinYMid"
+    >
+      <path d="M-20 9.35l-9.08 8.54-.86-.81 6.54-7.31H-40V8.12h16.6L-29.94.81l.86-.81L-20 8.51v.84zm20-.81L9.08 0l.86.81L3.4 8.12H20v1.65H3.4l6.54 7.31-.86.81L0 9.39v-.85z" />
+    </svg>
+  );
+
+}
```

Comment:
Assuming this is identical to ArrowRightStraight, can we do this with `transform: scaleX(-1)` or `transform: rotate(180deg)` in CSS instead?

Reply:
Ah, I had much fun trying to make `rotate` work due to having to set the origin prior to running the transform. It rotated around 0,0 relative to the parent element rather than any specific point of the SVG. Honestly, I got fed up with it and just wanted to move on.

Reply:
Applying `transform: scaleX(-1)` seems to work for me.

Reply:
Hm. I still see it move down as a result of applying `scaleX(-1)` which is the same behaviour I saw when trying the `rotate(180deg)`